silverback/runner.py

```diff
@@ -109,6 +110,38 @@ async def _event_task(self, task_data: TaskData):
         handle an event handler task for the given contract event
         """
 
+    async def _cron_task(self, task_data: list[TaskData]):
+        cron_jobs: dict[TaskData, CronSchedule] = dict()
+
+        for td in task_data:
+            if cron_schedule := td.labels.get("cron"):
+                cron_jobs[td] = CronSchedule(cron=cron_schedule)
+
+            else:
+                # NOTE: Shouldn't happen but just in case
+                logger.warning(f"TaskData missing `cron` label: {td}")
+
+        if cron_jobs_str := "\n- ".join(map(str, cron_jobs.values())):
+            logger.info(f"Cron Jobs:\n- {cron_jobs_str}")
+
+        while True:
+            current_time = utc_now()
+            if current_time.second < CRON_CHECK_SECONDS:
+                # Print out current time every minute
+                logger.info(f"Current Time: {current_time}")
+                # NOTE: In the absence of any cron jobs, we still print out the current time
```

Comment:
suggestion: make this debug. bit verbose and not really useful unless you're debugging.

Reply:
I was actually thinking this could be really useful to see for endusers, because they would have some live logging being created in scenarios where they are solely doing event log monitoring and nothing is really happening. Also very useful to know what the bot's local time is vs. the blockchain's, to see if there is some drift occuring